TvMaterialCatalog/gradle/libs.versions.toml

```diff
@@ -1,26 +1,30 @@
 [versions]
-agp = "8.3.1"
+agp = "8.3.2"
 coil-compose = "2.5.0"
+foundation = "1.7.0-beta07"
```

Comment:
We could remove this line as Compose 1.7.0 became stable and a part of Compose BOM.